internal/client/client.go

```diff
@@ -48,6 +48,20 @@ type TemporaryCredentials struct {
 	SessionToken    string `json:"session_token"`
 }
 
+type AccountType int
+
+const (
+	AccountTypeUnknown    AccountType = 0
```

Comment:
Geez, on second thought, `AccountTypeUnknown` isn't necessary. I was thinking in terms of languages with a fixed set of "enum" values.